packages/contact-center/station-login/src/station-login/station-login.presentational.tsx

```diff
@@ -109,54 +109,55 @@ const StationLoginPresentational: React.FunctionComponent<StationLoginPresentati
       <div className="box">
         <section className="section-box">
           <fieldset className="fieldset">
-            <legend className="legend-box">Agent</legend>
-            <div style={{display: 'flex', flexDirection: 'column', flexGrow: 1}}>
-              <div style={{display: 'flex', gap: '1rem'}}>
-                <fieldset
-                  style={{
-                    border: '1px solid #ccc',
-                    borderRadius: '5px',
-                    padding: '10px',
-                    marginBottom: '20px',
-                    flex: 0.69,
-                  }}
-                >
-                  <legend className="legend-box">Select Team</legend>
-                  <select id="teamsDropdown" className="select">
-                    Teams
-                  </select>
-                </fieldset>
-                <fieldset className="fieldset">
-                  <legend className="legend-box">Agent Login</legend>
-                  <select name="LoginOption" id="LoginOption" className="select" onChange={selectLoginOption}>
-                    <option value="" hidden>
-                      Choose Agent Login Option...
-                    </option>
-                  </select>
-                  <input
-                    className="input"
-                    id="dialNumber"
-                    name="dialNumber"
-                    placeholder="Extension/Dial Number"
-                    type="text"
-                    onInput={updateDN}
-                  />
-                  {isAgentLoggedIn ? (
-                    <ButtonPill id="logoutAgent" onPress={logout} color="cancel">
-                      Logout
-                    </ButtonPill>
-                  ) : (
-                    <ButtonPill id="AgentLogin" onPress={login} color="join">
-                      Login
-                    </ButtonPill>
-                  )}
-                </fieldset>
-              </div>
-            </div>
+            <Text tagName={'span'} type="heading-small-bold">
+              Set your interaction preferences
+            </Text>
           </fieldset>
+          <fieldset className="fieldset">
+            <Text tagName="span" type="body-large-regular">
+              Handle calls using
+            </Text>
+            <select name="LoginOption" id="LoginOption" className="select" onChange={selectLoginOption}>
+              <option value="" hidden>
+                Choose Agent Login Option...
+              </option>
+            </select>
```

Comment:
This still seems to be the native dropdown and doesn't come from MRV2. Could you please check this?